appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -87,22 +83,6 @@ private fun MainSurface() {
     }
 }
 
-@Composable
-private fun BoxScope.ComposeLastRenderingState(state: Rendering) {
-    val shakeGestureListener = LocalShakeGestureListener.current
-    val viewModel = LocalViewModel.current
-
-    LaunchedEffect(state.isPreview) {
-        if (state.isPreview) {
-            shakeGestureListener.addListener(true) { viewModel.refreshPreview() }
-        } else {
-            shakeGestureListener.clearListener()
-        }
-    }
-
-    ComposeContainer(state.stepContainer, state.position)
```

Comment:
moved this call inline to L68 since it was the only thing this function was left doing, after removing shake gesture stuff.